### PR TITLE
SG-11264 Add figure include for captioned images and provide example in documentation

### PR DESCRIPTION
**Pull request by willis102**
Add an include allowing advanced formatting of images, using the HTML5 `<figure>` tag.

### CODE REVIEW

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/776a30aa9a9e7e4262ce06ed7e2a57ca91d64d94/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/2f25149078fcfd626ceb8ab12877a8618367001e/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/b95203fc222ed1906ebec30fd62750057dab8dfe/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/6982a7bc340c4a6f194966a9167d102702df2465/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/edac8ffdd573e0c8f5ca23304b35b723c92f7ea8/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/1e486594d2d0d449741d631c322b9bac91b3845d/index.html)

**Comment by willis102** [comment]
This is ready to go, but since it depends on just-the-docs, it would probably be better to wait on [this PR](https://github.com/shotgunsoftware/just-the-docs/pull/2) on that repo, and update Gemfile.lock to point to the commit on master after the merge.

File: docs/en/authoring/figures.md [hunk 1]
```diff
@@ -23,3 +23,22 @@ When including images, figures, screenshots and diagrams, follow these guideline
...
+
+{% raw  %}
+```
+{% include figure src="../../images/landing-page/dev_icon.png" caption="An example figure." width="100px" %}
+```
+{% endraw  %}
+
+Supported variables include:
+  - `src`: The path to the image
+  - `caption`: An optional image caption.  If omitted, the `<figcaption>` element will be omitted.
```
Comment (manneohrstrom) [inline docs/en/authoring/figures.md]: i don't think we need to mention `<figcaption>`. Maybe just skip the second sentence?

**Review by manneohrstrom — APPROVED** [review]
Looks great! see nitpicky comment about the figcaption :)

File: docs/en/authoring/figures.md (hunk 1 above)
Comment (willis102) [reply docs/en/authoring/figures.md]: Makes sense, that's probably unnecessary and could be confusing from a less-technical user's point of view.

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/fb0bf99e8427e5fd918bcae7bf792cf907a6223d/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/a3bfe3d67258c454bcac391015737f3e46963c6f/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/b96084060ab2ddeb5dc1886ff36506fe6b369bc2/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/e8f1596f59cb64522f4b391cc83ed3d45ced4fd1/index.html)

**Pull request merged**